bindata/v4.0.0/apiservice-cabundle-controller/clusterrole.yaml

```diff
@@ -23,9 +23,3 @@ rules:
   - watch
   - update
   - create
```

Comment:
I know this is preexisting, but I do not think this needs access to config maps at all.

Reply:
a84d0954